Samples/Islands/cs-wpf-packaged/WpfWithIslandApp/WpfWithIslandApp/WpfWithIslandApp.csproj

```diff
@@ -4,9 +4,9 @@
     <OutputType>WinExe</OutputType>
 
     <!--
-      NOTE: Normally in net8 one wouldn't set a speficic Windows version, but
-      If we leave this as "net8.0-windows", we get an error because TargetPlatformVersion
-      is not set in this file: Microsoft.InteractiveExperiences.Common.targets
+      Changed_from_original_app
+      The lowest version of Windows that WinAppSDK supports is 17763.
+      But Lottie requires Windows 10.0.19041.0 or higher, so we target this one.
```

Comment:
@JesseCol, WinAppSDK Lottie only requires RS5 (the same as the rest of WASDK). Let's follow-up on this one.